cob_sound/ros/src/sound.cpp

```diff
@@ -41,6 +41,7 @@ class SoundAction
     as_say_(nh_, "say", boost::bind(&SoundAction::as_cb_say_, this, _1), false),
     as_play_(nh_, "play", false)
   {
+    nh_ = ros::NodeHandle("~");
```

Comment:
This will modify the namespace of the sound services and topic, right?
Will this have an effect on simple script server and other packages?